extensions/vscode/src/language-server/languagePlugin.ts

```diff
@@ -0,0 +1,100 @@
+import { CodeMapping, type LanguagePlugin, type VirtualCode } from '@volar/language-core';
+import type * as ts from 'typescript';
+import type { URI } from 'vscode-uri';
+
+export const frontmatterPlugin = (debug: (message: string) => void) =>
```

Comment:
I actually wanted to add this to our eslint plugin too but generally our convention is to have function bodies which make it a lot easier to debug functions and you can more quickly add `debugger` statements.

Reply:
I would also make this a `function`, as it is really a Function.